### PR TITLE
Fixed notification overflow in IE11

### PR DESCRIPTION
**Pull request by boromo**
### Prerequisites

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### Description

Basically fix for a fix, since the fix from the previous version introduced unwanted behavior - it was limiting the notification component to the fixed width of 245px

### CODE REVIEW

**Comment by kotielnikov** [comment]
run `npm run test:visual -- -u` to update snapshots
